manifests/params.pp

```diff
@@ -116,6 +116,6 @@
   $puppi_helper = 'standard'
   $debug = false
   $audit_only = false
-  $noops = undef
+  $noops = false
```

Comment:
I'll leave this up to you, I would assume that the meaning of the 'noop' parameter on resource level is a bit different then the 'noop' on command line. So, in --noop I just want to know that my manifests are in noop mode due to the variable. As you said, once the parameter gets removed, we should be all set. Similar issue would apply for the logrotate module (PR has been submitted in parallel to this one, too)